hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/SubscriptionHealthCheckProperties.java

```diff
@@ -0,0 +1,25 @@
+package pl.allegro.tech.hermes.management.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("subscription-health-check")
+public class SubscriptionHealthCheckProperties {
+    private long subscriptionHealthCheckTimeoutMillis = 8000;
+    private int subscriptionServiceThreads = 16;
```

Comment:
For now the configuration file will look like this:
```
subscription-health-check:
  subscription-health-check-timeout-millis: 8000
  subscription-service-threads: 16
```
It seems a bit verbose. What do you think about simpler names like these:
```
subscription-health-check:
  timeout-millis: 8000
  threads: 16
```